.circleci/config.yml

```diff
@@ -588,9 +598,12 @@ jobs:
       - notify:
           message: "Release process for the desktop app ended, the new release can be found on [GitHub](https://github.com/mattermost/desktop/releases)."
 
-
 workflows:
   version: 2
+  sec:
```

Comment:
What's this bit for? Just a test or something?

Reply:
Yep . The whole file will be removed after we test that everything is in place

Reply:
> Looking good so far! One question: are we able to download the builds from the CI build itself like we could on CircleCI?
> 
> The scheduling of the nightly build looks good to me. Let me know what help I can provide for the channel notifications, I thought we just needed to hit a webhook on community.

@devinbinnie Let's sync together on this . There is a difference on how Github handles artifacts vs CirlceCI so I would like your input